test/unit/select_filters_test.lua

```diff
@@ -311,7 +311,7 @@ g.test_unicode_collation = function()
             types = {'string', 'string', 'string', 'number'},
             early_exit_is_possible = false,
             values_opts = {
-                {collation='unicode_ci'},
+                {collation='unicode'},
                 {collation='unicode_ci'},
                 {collation='unicode_ci'},
```

Comment:
Maybe here you could add "binary" and "none" collactions?

Reply:
I would like to add one more test for this. It's too much for one test.

Reply:
Sounds good